docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ services:
     networks:
       - eventsgateway
   kafka:
-    image: wurstmeister/kafka:1.0.0
+    image: wurstmeister/kafka:2.12-2.2.0
```

Comment:
We're using kafka 2 in MSK?

Reply:
MSK is 2.1.0. I've downgraded from 2.2.0 to 2.1.1.